tests/integration/vector_io/test_openai_vector_stores.py

```diff
@@ -55,6 +55,7 @@ def skip_if_provider_doesnt_support_openai_vector_stores_search(client_with_mode
         "keyword": [
             "inline::sqlite-vec",
             "remote::milvus",
+            "inline::milvus",
```

Comment:
Just a quick question for my own understanding, how is it possible to have currently (in main) **hybrid** with `inline::milvus` in the tests, but not **keyword** with `inline::milvus`? I thought Hybrid meant a combination of Vector and Keyword search.

Reply:
The same thought actually made me look into Milvus-lite repo, and ended up finding the PR from their end. Just didn't want to pause hybrid implementation PR to get merged, so ended up creating a new one for this. 
But both keyword and hybrid implementations were included in the specific pymilvus version which we had pinned, which is why it worked out of the box.

Reply:
That's awesome, thanks for explaining! :)